public_html/wp-content/mu-plugins/blocks/assets/src/blocks.js

```diff
@@ -6,10 +6,14 @@ const { registerBlockType } = wp.blocks;
 /**
```

Comment:
`SHOULD` - When this is committed to SVN, `#33` will get linked to https://meta.trac.wordpress.org/ticket/33. You could instead use the full PR URL.

`MIGHT` - I don't know that we even need to reference the PR, though, since it doesn't really make sense in an SVN context, so I'd probably just remove it. If we were committing directly to Git, though, it'd definitely be good to add something like that.

Reply:
Its already expanded, I think GitHub UI is showing it this way. (Take a look at the text in git log in your terminal)